app/email.go

```diff
@@ -781,6 +781,8 @@ func (es *EmailService) SendPaymentFailedEmail(email string, locale string, fail
 	bodyPage.Props["Button"] = T("api.templates.over_limit_fix_now")
 	bodyPage.Props["EmailUs"] = T("api.templates.email_us_anytime_at")
 
+	bodyPage.Props["Footer"] = T("api.templates.copyright")
```

Comment:
Do we need to make thing change to the overage 7/14/30 days sequence emails as well?

Reply:
This is already in those emails. I just forgot to add it to these 2